### DISCUSSION TITLE
add support to change what the vintage shortcut key does

### DISCUSSION THREAD

**Discussion by m70v** [BartoszCichecki/LenovoLegionToolkit#1437] [Ideas]
so generally legions have this button that looks like a star usually located on the F9 key. what it does is that it opens lenovo vintage when pressed. what i want is the ability to change the functionality of this key to open other apps.
thank you!

**Comment by BartoszCichecki** [comment]
This feature is already there.

**Reply by m70v** [reply]
can you please tell me where i can find it?

**Reply by BartoszCichecki** [reply]
In Settings you can assign a Quick Action to single or double press of a Smart Key (Fn+F9).

**Reply by m70v** [reply]
it wasn't possible to enable it directly since i had to disable an option but thank you, really appreciate the effort put onto this app and hope you continue this amazing work!!
